### ISSUE TITLE
Gardener upgrade to v1.51.0 fails after minimum Kubernetes requirement was raised

### ISSUE THREAD

**Issue by namsral** [23technologies/23ke-charts#304]
The deployment gardener-controller-manager fails after upgrading to Gardener v1.51.0 with the following error:

```
{"level":"info","ts":"2022-07-27T13:52:44.310Z","msg":"Starting gardener-controller-manager","version":"v1.51.0"}
{"level":"info","ts":"2022-07-27T13:52:44.311Z","msg":"Feature Gates","featureGates":""}
{"level":"info","ts":"2022-07-27T13:52:44.313Z","msg":"maxprocs: Leaving GOMAXPROCS=8: CPU quota undefined"}
{"level":"info","ts":"2022-07-27T13:52:44.332Z","logger":"clientmap","msg":"Created new ClientSet","clientmap":"GardenClientMap","key":"garden","hash":""}
I0727 13:52:44.336501       1 leaderelection.go:248] attempting to acquire leader lease garden/gardener-controller-manager-leader-election...
{"level":"info","ts":"2022-07-27T13:52:44.336Z","logger":"server","msg":"Starting new HTTP server","listenAddress":"0.0.0.0:2718"}
I0727 13:52:44.344684       1 leaderelection.go:258] successfully acquired lease garden/gardener-controller-manager-leader-election
{"level":"info","ts":"2022-07-27T13:52:44.344Z","msg":"Acquired leadership, starting controllers"}
{"level":"error","ts":"2022-07-27T13:52:44.661Z","msg":"Failed to run controllers","error":"failed bootstrapping garden cluster: the Kubernetes version of the Garden cluster must be at least 1.20","stacktrace":"runtime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1571"}
{"level":"info","ts":"2022-07-27T13:52:44.661Z","msg":"Lost leadership, terminating"}
```

Gardener is deployed with the default kube-apiserver images, see [values.yaml][1]:

```sh
% kubectl -n garden get deployments garden-kube-apiserver -o json | jq -r '.spec.template.spec.containers[] | .image'
k8s.gcr.io/kube-apiserver:v1.19.15
k8s.gcr.io/kube-controller-manager:v1.19.15
```

The minimum Kubernetes requirement was raised to 1.20 in https://github.com/gardener/gardener/pull/6255.

[1]: https://github.com/23technologies/23ke-charts/blob/5ade1f80be5894737a73901df0afee86cc493677/charts/garden-kube-apiserver/values.yaml

**Comment by berendt** [comment]
We're already working on a solution for this one. Please have a look at https://github.com/23technologies/23ke-charts/tree/kube-apiserver. I think Malte will merge this one at the beginning of the next week.

**Comment by mxmxchere** [comment]
Exactly, as far as we have tested gardener v1.51+ works fine now with the new api-server chart. I'll close this issue, please report if there are any issues and we need to check again.

**Comment by namsral** [comment]
The change in syntax for the _garden-kube-apiserver_ chart version confused my version tracker.

The previous version was `3.32.0` similar to the garden-etc chart, the latest version is `v1.24.3`.

**Comment by mxmxchere** [comment]
Hi Lars, i think there are two different issues:

1st: **v vs. no-v in the version number**: Helm requires strict semver (no v-prefix) and we will follow that in the future.

2nd: 1.24.3 is smaller than 3.32 -> we will continue with the 4.x series and have released the same apiserver with the 4.0.0 version number
